src/angularjs/LineChart.ts

```diff
@@ -105,7 +105,7 @@ module n3Charts {
       };
 
       scope.$watch('options', updateAll, true);
-      scope.$watch('data', updateData, scope.options.deepWatchData);
+      scope.$watch('data', updateData, options ? options.deepWatchData : true);
```

Comment:
this assumes that `Options.Options` the same thing as `scope.options`. is that true?